Makefile

```diff
@@ -383,6 +394,54 @@ depend:
 	@echo "Updating Helm dependencies (for $(MINIO_CHART))..."
 	@cd deploy/helm && helm dependency update $(MINIO_CHART_PATH) || exit 1
 
+	@echo "Updating Helm dependencies (for aiobs-app)..."
+	@cd deploy/helm && helm dependency update aiobs-app || exit 1
+
+	@echo "Updating Helm dependencies (for aiobs-infra)..."
+	@cd deploy/helm && helm dependency update aiobs-infra || exit 1
+
+	@echo "Updating Helm dependencies (for aiobs-stack)..."
+	@cd deploy/helm && helm dependency update aiobs-stack || exit 1
+
+
+# Install full stack using umbrella chart
+.PHONY: install-stack
```

Comment:
Are those install/unstall targets (such install-mcp-server etc) in the Makefile continue to work?

Reply:
For individual will helpfull for more development  and troubleshooting purpose 
these new helm umbrella chart targets mostly required for coming next operator support